CONTRIBUTING.rst

```diff
@@ -523,8 +509,7 @@ Benchmarks
 ----------
 While not mandatory for most pull requests, we ask that performance related
 PRs include a benchmark in order to clearly depict the use-case that is being
```

Comment:
Unrelated to this PR, but I would write "performance-related" (hyphen) and "use case" (no hyphen) :upside_down_face:

Reply:
I was just laughing about us thinking about using `use case`, and how the first Google result then only discusses alternatives!